CONTRIBUTING.rst

```diff
@@ -29,86 +29,80 @@ Documentation Convention
-|           | Use “A relation between two concepts where…” as the       |
-|           | sentence pattern if possible.                             |
-+-----------+-----------------------------------------------------------+
-| Example   | Example shows how words are linked in this relation, and  |
-|           | each word will also be linked to the lexicon so the user  |
-|           | can check the details of that word directly. 1. At least  |
-|           | 2 examples. If there is no example in the docs, go to     |
-|           | related resources or wiki to find more. 2. Link the       |
-|           | examples to entries in the OMW. Go to below two websites  |
-|           | to search for the right sense of an example word: -       |
-|           | http://compling.hss.ntu.edu.sg/omw/cgi-bin/wn-gridx.cgi - |
-|           | https://lr.soh.ntu.edu.sg/omw/omw                         |
-|           | relations.hypernym.exe.en = """ \* \* <sense:             |
-|           | pwn-3.0:07649854-n:meat> \* ``hypernym`` \* <sense:       |
-|           | pwn-3.0:07663592-n:beef> \* \* *edible fruit*             |
-|           | ``hypernym`` *pear* \* *wordbook* ``hypernym``            |
-|           | *dictionary* """ `Meat`_ `Beef`_                          |
-+-----------+-----------------------------------------------------------+
-| Test      | The Test provide a method to justify whether the          |
-|           | linguistic data concords with this relation type or not.  |
-|           | If there is no test in the docs, go to EuroWordnet        |
-|           | general document to see whether there are any EWN tests   |
-|           | there.                                                    |
-+-----------+-----------------------------------------------------------+
-| XML       | XML sample provides the sample code of relations.         |
-| Sample    |                                                           |
-+-----------+-----------------------------------------------------------+
-| Comment   | Note provides some additional info the user may need to   |
-+-----------+-----------------------------------------------------------+
-
-.. _Meat: https://lr.soh.ntu.edu.sg/omw/ili/concepts/77100
-.. _Beef: https://lr.soh.ntu.edu.sg/omw/ili/concepts/77197
-
-
-**Other name in specific Project**
+- **Relation Name** :  The name of the relation
+
+- **Short Definition** : The short definition only shows up when the user’s mouse hover over the relation name, it is a brief introduction of the relation.
+
+1. Use concept A / concept B (A / B) for reference, DO NOT use concept X / concept Y (X /y).
+
+2. Use “A is … of B” as the sentence pattern if possible.
+
+- **Short Example** : The short example only shows up when the user’s mouse hover over the relation name, it just shows the examples without links to lexicon. At least 2 examples. If there is no example in the docs, go to related resources or wiki to find more.
```

Comment:
Ditto about "mouse hover". And what are the related resources or wiki?

Reply:
Corrected.